src/App.js

```diff
@@ -2,14 +2,21 @@ import "./App.css";
 import { WelcomeUser } from "./pages/WelcomeUser";
 import { UserOnboarding } from "./components/UserOnboarding";
 import { useApp } from "./context/app-context";
+import { useEffect, useState } from "react";
+import { imageDb } from "./db/imageDb";
 
 function App() {
   const { isUser } = useApp();
+  const [background, setBackground] = useState();
+
+  useEffect(() => {
```

Comment:
Do we need a useEffect here? Because on every page refresh new math.random is generated & hence a background image as well

Reply:
the thing is it was changing every second, so what I did  I used empty dependency array to run it only on initial render, that's why I have used it.